spec/controllers/coronavirus_form/essential_supplies_controller_spec.rb

```diff
@@ -46,7 +50,10 @@
     end
 
     it "clears session values for later food questions for a yes response" do
+      session[:dietary_requirements] = "Yes"
+      session[:carry_supplies] = "Yes"
```

Comment:
Thanks for adding this. Minor nitpick is that this change should probably be fixed up.

Reply:
Ah, yes, didn't see that I'd added this test in an earlier commit.  Have fixed.